packages/desktop-electron/menu.js

```diff
@@ -1,6 +1,6 @@
 const { Menu, ipcMain, app, shell } = require('electron');
 
-function getMenu(isDev, createWindow) {
+function getMenu(isDev, createWindow, budgetId = undefined) {
```

Comment:
Passing budgetId here, defaulting to undefined. If passed as undefined we're not on a budget (probably the budget manager screen)